src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,40 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Encapsulates the information for an Event object that contains the description, completion status and due date.
+ */
+public class Event extends Task {
+    public static final String TAG = "E";
+    private LocalDate at;
```

Comment:
Maybe the name can be more descriptive.

Reply:
Again, not coding violation but I do agree with Siddanth. The name can be more descriptive or you can add comments to better describe this

Reply:
Thank you. Will improve on it in the next commit.